### ISSUE TITLE
Lags - background delay

### ISSUE THREAD

**Issue by KateCee11** [valinet/ExplorerPatcher#1160]
Since installing my typing is stalled (letters catch up) and cursor blinks the hourglass for a second or two. I didn't see this in other issue or discussions.  Has this happened, and is there a setting or fix ?

Thanks.... Kate

**Comment by jozjan** [comment]
I had similar issues with lagy windows, slow typing, slow screen rendering, etc. All those started to appear only after a few mins or an hour after fresh a Windows start. 
Today I uninstalled ExplorerPatcher, and I am trying to validate if the same lagy Windows behavior is still present on my system.

**Comment by KateCee11** [comment]
I had more RAM installed in my laptop and that seems to have stopped the lag.  I actually didn't notice it after the first few days, and didn't do anything to resolve the problem.  I will keep an eye on it, of course.

**Comment by jozjan** [comment]
Update: although without EP the Win 11 lagginess is not as severe as before, it's still there :-/ It seems the root cause wasn't EP.

**Comment by Poopooracoocoo** [comment]
Judging by the last two comments I think I can close this. Feel free to leave a comment and I'll reopen, or open a new issue or discussion. Make sure to include your OS build and EP version.
